provisions/roles/repo_tracking/tasks/main.yml

```diff
@@ -4,8 +4,13 @@
   sudo: yes
   tags: repo_tracking
 
+- name: Install pip
+  yum: name=python-pip state=installed
+  sudo: yes
+  tags: repo_tracking
+
```

Comment:
We are anyway installing these pip, django, etc for 3 nodes including jenkins_master with
https://github.com/CentOS/container-pipeline-service/pull/302/commits/0a60e279ac26f9bc6e10df5a22b099eaef118b56

We dont need this part of code in this commit.